ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -202,6 +213,52 @@ access_lists:
         action: "< action as string >"
 ```
 
+Improved data model has a more sophisticated design documented below:
+
+```yaml
+ip_access_lists:
+  - name: "< access list name as string >"
+    counters_per_entry: < true | false >
+    entries:
+      # remark entry
+      - sequence: < acl entry sequence number >  # optional
+        # NOTE: if remark is defined, other keys in acl entry will be ignored
+        remark: "< Comment, up to 100 characters >"
+      # normal entry
+      - sequence: < acl entry sequence number >  # optional
+        action: "< permit | deny >"  # required
+        protocol: "< ip | tcp | udp | icmp | other protocol name or number >"  # required
+        # NOTE: A.B.C.D without a mask means host
+        source: "< any | A.B.C.D/E | A.B.C.D >"  # required
+        source_ports_match: "< eq | gt | lt | neq | range >"  # eq is default
+        source_ports: ["< tcp/udp port name or number >",]  # optional
+        # NOTE: A.B.C.D without a mask means host
+        destination: "< any | A.B.C.D/E | A.B.C.D >"  # required
+        destination_ports_match: "< eq | gt | lt | neq | range >"  # eq is default
+        destination_ports: ["< tcp/udp port name or number >",]  # optional
+        tcp_flags: ["< tcp flag name >",]  # optional
+        fragments: < true | false >  # optional, match non-head fragment packets
+        log: < true | false >  # optional, log matches against this rule
+        ttl: < <0-254> TTL value >  # optional
+        ttl_match: "< eq | gt | lt | neq >"  # optional
+        icmp_type: "< Message type name/number for ICMP packets >"  # optional
+        icmp_code: "< Message code for ICMP packets >"  # optional
+        nexthop_group: "< nexthop-group name >"  # optional
+        tracked: < true | false > # optional, match packets in existing ICMP/UDP/TCP connections
+        dscp: "< DSCP value or name >"  # optional
+        vlan_number: < vlan number >  # optional
+        vlan_inner: < true | false >  # optional, default - false
+        vlan_mask: "< 0x000-0xFFF  Vlan mask >"  # optional
+```
+
+The improved data model allows to define the maximum number of ACL entries that AVD is allowed to generate.
+If the number is above the limit, the playbook will fail. That provides a simplified control over hardware utilisation.
+The numbers must be based on the hardware tests and AVD does not provide any guidance.
+
+```yaml
+ip_access_lists_max_acle: <maximum number of ACL entries allowed per switch>  # default is 10000
```

Comment:
please don't set a default here.
The first sentence in the text is not clear. Suggestion: "`ip_access_lists_max_acle` can be used to limit the number of ACL entries that AVD is allowed to generate. Only entries under `ip_access_lists` will be counted. If the number is above the limit, the playbook will fail. This provides a simplified control over hardware utilization.
The numbers must be based on the hardware tests and AVD does not provide any guidance. Note that other EOS features may use the same hardware resources and affect the supported scale"

Reply:
Resolved. Inventing the correct default was an impossible task anyway, removing it an making the var optional making the logic a lot more straightforward.